pdns/recursordist/docs/settings.rst

```diff
@@ -750,6 +750,21 @@ Maximum number of DNS queries in a TCP connection.
 
 Total maximum number of milliseconds of wallclock time the server may use to answer a single query.
 
+.. _setting-max-udp-queries-per-round:
+
+``max-udp-queries-per-round``
+----------------------------------
+.. versionadded:: 4.2.0
```

Comment:
Should be 4.1.4, I guess?

Reply:
These docs are not used. In de changelog PR I have changed this.

Reply:
Oh right, everything looks good then :-)